src/Str.php

```diff
@@ -40,7 +44,7 @@ public static function hasSuffix(string $value, string $suffix): bool
      */
     public static function addSuffix(string $value, string $suffix): string
     {
-        return self::removeSuffix($value, $suffix).$suffix;
+        return self::removeSuffix($value, $suffix) . $suffix;
```

Comment:
This can be updated as `return u($value)->ensureEnd($suffix);`

Reply:
test failed => `addSuffix('Generatecommand', 'Command')` produce `GeneratecommandCommand` instead of `GenerateCommand`